packages/functions/lib/index.d.ts

```diff
@@ -144,6 +144,28 @@ export namespace FirebaseFunctionsTypes {
     (data?: any): Promise<HttpsCallableResult>;
   }
 
+  /**
+   * An HttpsCallableOptions object that can be passed as the second argument to `firebase.functions().httpsCallable(name, HttpsCallableOptions)`.
+   * It allows you to control how long the application will wait for the cloud function to respond in milliseconds.
+   *
+   * ### Example
+   *
+   * ```js
+   * const instance = firebase.functions().httpsCallable('order', { timeout: 7000 });
+   * // The below will wait 7 seconds for a response from the cloud function before an error is thrown
+   * try {
+   *  const response = await instance({
+   *    id: '12345',
+   *  });
+   * } catch (e) {
+   *  console.log(e);
+   * }
+   * ```
+   */
```

Comment:
Can we move this docs/example and the description about the timeout option to the timeout property, rather than on the `HttpsCallableOptions` interface?